site/tutorials/snippets/09/monitoring/setupDynatrace.md

```diff
@@ -9,75 +9,23 @@ Positive
-  - Dynatrace SaaS tenant (this format is most likely for you): `{your-environment-id}.live.dynatrace.com`
-  - Dynatrace-managed tenant: `{your-domain}/e/{your-environment-id}`
-
-    If running on a Unix/Linux based system, you can use variables for ease of use. Naturally, it is also fine to just replace the values in the `kubectl` command itself.
-
-    <!-- var DT_TENANT -->
-    <!-- var DT_API_TOKEN -->
-    <!-- var DT_PAAS_TOKEN -->
-
-    ```
-    export DT_TENANT=yourtenant.live.dynatrace.com
-    export DT_API_TOKEN=yourAPItoken
-    export DT_PAAS_TOKEN=yourPAAStoken
-    ```
-
-    Negative
-    : Please make sure your DT_TENANT does _not contain_ any trailing slashes nor a https:// in the beginning.
-
-    If you used the variables, the next command can be copied and pasted without modifications. If you have not set the variables, please make sure to set the right values in the next command.
-    
-    <!-- command -->
-    ```
-    kubectl -n keptn create secret generic dynatrace --from-literal="DT_TENANT=$DT_TENANT" --from-literal="DT_API_TOKEN=$DT_API_TOKEN"  --from-literal="DT_PAAS_TOKEN=$DT_PAAS_TOKEN" --from-literal="KEPTN_API_URL=http://$(kubectl -n keptn get ingress api-keptn-ingress -ojsonpath='{.spec.rules[0].host}')/api" --from-literal="KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n keptn -ojsonpath='{.data.keptn-api-token}' | base64 --decode)" --from-literal="KEPTN_BRIDGE_URL=http://$(kubectl -n keptn get ingress api-keptn-ingress -ojsonpath='{.spec.rules[0].host}')/bridge" 
-    ```
-
 ## Deploy Dynatrace OneAgent Operator
 
 To monitor a Kubernetes environment using Dynatrace, please setup dynatrace-operator as described below, or visit the [official Dynatrace documentation](https://www.dynatrace.com/support/help/technology-support/cloud-platforms/kubernetes/deploy-oneagent-k8/).
 
 For setting up dynatrace-operator, perform the following steps:
 
 1. Log into your Dynatrace environment
-1. Open Dynatrace Hub (on the left hand side, scroll down to **Manage** and click on **Deploy Dynatrace**)
+1. Open Dynatrace Hub (on the left hand side, scroll down to **Manage** and click on **Hub**)
 1. Within Dynatrace Hub, search for Kubernetes
    ![Dynatrace Hub](./assets/dt-hub-kubernetes.png)
 1. Click on Kubernetes, and select **Monitor Kubernetes** at the bottom of the screen
-1. In the following screen, select the Platform, a PaaS and API Token, and the OneAgent installation options.
-
-   **Note**: Please make sure to tick *Enable volume storage* if you are on GKE, Anthos, CaaS and PKS.
+1. In the following screen, select the Platform and click on **Create tokens** to generate PaaS and API tokens.
+1. Please make sure to tick *Enable volume storage* if you are on GKE, Anthos, CaaS and PKS.
 
    ![Dynatrace Kubernetes Monitoring](./assets/dt-kubernetes-monitor.png)
```

Comment:
I had issues on my GKE Test cluster with the SSL Check, perhaps a note would be good like to "Skip SSL Certificate check" for the API endpoint